pkg/auth/uaa/tanzu.go

```diff
@@ -4,20 +4,28 @@
 package uaa
 
 import (
+	"os"
+	"strconv"
+
 	"golang.org/x/term"
 
 	"github.com/vmware-tanzu/tanzu-plugin-runtime/config"
 
 	"github.com/vmware-tanzu/tanzu-cli/pkg/auth/common"
+	"github.com/vmware-tanzu/tanzu-cli/pkg/constants"
 )
 
 const (
 	// Tanzu CLI client ID for UAA that has http://127.0.0.1/callback as the
 	// only allowed Redirect URI and does not have an associated client secret.
-	tanzuCLIClientID     = "tp_cli_app"
-	tanzuCLIClientSecret = ""
-	defaultListenAddress = "127.0.0.1:0"
-	defaultCallbackPath  = "/callback"
+	tanzuCLIClientID = "tp_cli_app"
+	// Alternate client ID for UAA associated with a longer refresh token
+	// validity. Use this for CLI use cases where it is impractical to
+	// interactively reauthenticate once the refresh token expires.
+	tanzuCLIClientIDExtended = "tp_cli_app_ext"
```

Comment:
Do we need to set this to the same as the other one for now?

Reply:
Ok, I saw on chat that you will leave the logic to fallback to the `TANZU_CLI_USE_ALTERNATE_UAA_CLIENT` variable but that we won't use it for now